tests/e2e/validator.go

```diff
@@ -129,9 +131,8 @@ func (v *validator) createConsensusKey() error {
 }
 
 func (v *validator) createKeyFromMnemonic(name, mnemonic string) error {
-	cdc := params.MakeTestEncodingConfig().Codec
```

Comment:
here is `cdc := gaia.MakeTestEncodingConfig().Codec`

Reply:
@alexanderbez noted that this should be removed in light of a globally available `cdc`. Unless was this the culprit of the keys encoding issue?

Reply:
hi @okwme , i think yours is right!